metecho/api/sf_org_changes.py

```diff
@@ -221,3 +221,18 @@ def compare_revisions(old_revision, new_revision):
             if new_revision[mt][mn] > old_revision.get(mt, {}).get(mn, -1):
                 ret[mt].append(mn)
     return ret
+
+
+def get_org_config(*, user, repo_id, project_path):
+    repository = get_repo_info(user, repo_id=repo_id)
+    branch = repository.default_branch
+    cci = BaseCumulusCI(
+        repo_info={
+            "root": project_path,
+            "url": repository.html_url,
+            "name": repository.name,
+            "owner": repository.owner.login,
+            "commit": branch,
+        }
+    )
+    return cci.project_config
```

Comment:
I don't actually know how it differs, because I am not sure of the subtleties of difference between `BaseCumulusCI`'s `project_config` and the `GlobalConfig`'s `project_config`. There's a lot of stuff like that in CumulusCI that I remain unclear on.

Reply:
@wlonk Agreed -- did you see the linked explanation here? https://github.com/oddbird/Metecho/pull/191#discussion_r386451226

Reply:
Does this help?
- GlobalConfig: represents the default ("global") cumulusci.yml from CumulusCI itself. Does NOT represent any particular project; the `get_project_config` method is used internally by CumulusCI when loading cross-project sources that are referenced by a ProjectConfig. Metecho overrides it to make sure the sub-project config is constructed using the correct class
- ProjectConfig: represents the cumulusci.yml for a particular project
- BaseCumulusCI: a wrapper that provides access to both the global config and project config objects, and a convenient way to construct them both at once.

Reply:
Actually I'm wrong about GlobalConfig.get_project_config. It is no longer used by CumulusCI and should be removed from Metecho as well.

Reply:
Well that's a helpful change! "Don't use one anymore"!

Reply:
@wlonk GlobalConfig.get_project_config is currently used within Metecho by get_project_config in gh.py, which can be updated to just directly construct a ProjectConfig

Reply:
@wlonk Why is this method still useful? It seems duplicative with `gh.get_project_config`.

Reply:
Right, thanks.